ankh/main.go

```diff
@@ -222,14 +223,14 @@ func execute(ctx *ankh.ExecutionContext) {
 
 func main() {
 	app := cli.App("ankh", "Another Kubernetes Helper")
-	app.Spec = "[-v] [--config] [--kubeconfig] [--datadir] [--context] [--set...] [--filter]"
+	app.Spec = "[-v] [--ankhconfig] [--kubeconfig] [--datadir] [--context] [--set...] [--filter]"
```

Comment:
i made a poor decision in an earlier PR to rename `ankhconfig` to `config` - this reverts that decision